src/main/java/racingcar/model/RaceCount.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.model;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+import racingcar.constants.ErrorMessage;
+
+public class RaceCount {
+    private static final Pattern RACE_TIMES_PATTERN = Pattern.compile("^[0-9]+$");
+
+    private final int raceCount;
+
+    public RaceCount(String raceCount) {
+        hasCharacters(raceCount);
+        isBlank(raceCount);
+        isZero(raceCount);
+        this.raceCount = Integer.parseInt(raceCount);
+    }
+
+    public static void hasCharacters(String raceCount) {
+        if (!RACE_TIMES_PATTERN.matcher(raceCount).matches()) {
+            throw new IllegalArgumentException(ErrorMessage.RACE_COUNT_MUST_NOT_HAVE_CHARACTER.getErrorMessage());
+        }
+    }
+
+    public static void isBlank(String raceCount) {
+        if (raceCount == null || raceCount.isBlank()) {
+            throw new IllegalArgumentException(ErrorMessage.RACE_COUNT_CAN_NOT_BE_EMPTY.getErrorMessage());
+        }
+    }
+
+    public static void isZero(String raceCount) {
+        if (raceCount.equals("0")) {
+            throw new IllegalArgumentException(ErrorMessage.RACE_COUNT_CAN_NOT_BE_ZERO.getErrorMessage());
```

Comment:
static import 를 사용해도 괜찮지 않을까요?
view 의 메서드를 사용하는 부분도 동일한 의견입니다!

```suggestion
import static racingcar.constants.ErrorMessage.RACE_COUNT_CAN_NOT_BE_EMPTY;
import static racingcar.constants.ErrorMessage.RACE_COUNT_CAN_NOT_BE_ZERO;
import static racingcar.constants.ErrorMessage.RACE_COUNT_MUST_NOT_HAVE_CHARACTER;
```